src/mango/src/mango_idx_view.erl

```diff
@@ -26,14 +26,20 @@
 
     indexable_fields/1,
     field_ranges/1,
-    field_ranges/2
+    field_ranges/2,
+
+    covers/2
 ]).
 
 -include_lib("couch/include/couch_db.hrl").
 -include("mango.hrl").
 -include("mango_idx.hrl").
 -include("mango_idx_view.hrl").
 
+-ifdef(TEST).
+-import(test_util, [as_selector/1]).
+-endif.
+
```

Comment:
The purpose of this `import` is to be able to share the `test_util:as_selector/1` function between multiple modules.  Initially, I added almost the same definition for multiple modules and then I merged them into a single one and placed in `test_util`.  Or are you talking about a local alias for `test_util:as_selector/1`?  In that case, it would not be that much of a gain, because its (proposed) name would not be that much shorter.

Reply:
The idea is that we've been removing them over the years as they are  not that common and make it a bit confusing to see where the functions come from. Since the `as_selector` in the test is composed with `indexable_fields(as_selector(Selector))` a dozen times or so, we could create a smaller function like

```erlang
 indexable(Selector) ->
      indexable_fields(as_selector(Selector)).
```

Then the test functions could be shorter. For instance:

```erlang
indexable_fields_type_test() ->
    ?assertEqual([], indexable(#{<<"field">> => #{<<"$type">> => undefined}}).
```

Reply:
Okay, thanks for the clarification and the hint!  There is already a function defined with the name `indexable` in this module, so I picked a different name.